proto/iotics/api/feed.proto

```diff
@@ -45,24 +45,24 @@ service FeedAPI {
 
 // A feed representation.
 message Feed {
```

Comment:
same as per Twin comment above

FeedIdentifier {
  string twinID
  string hostTwinID
  string id
}

Reply:
or alternatives

--- 

but I would question whether we need this message at all